pkg/services/ppmshipment/ppm_estimator.go

```diff
@@ -242,6 +245,53 @@ func (f *estimatePPM) estimateIncentive(appCtx appcontext.AppContext, oldPPMShip
 	return estimatedIncentive, estimatedSITCost, nil
 }
 
+func (f *estimatePPM) maxIncentive(appCtx appcontext.AppContext, oldPPMShipment models.PPMShipment, newPPMShipment *models.PPMShipment, checks ...ppmShipmentValidator) (*unit.Cents, error) {
+	// if the PPM shipment is past closeout then we should not calculate the max incentive, it is already set in stone
+	if newPPMShipment.Status == models.PPMShipmentStatusCloseoutComplete || newPPMShipment.Status == models.PPMShipmentStatusComplete {
+		return oldPPMShipment.MaxIncentive, nil
+	}
```

Comment:
Nil check suggestion (Actively reviewing, commenting this just in case of discussion)
```suggestion
	// if the PPM shipment is past closeout then we should not calculate the max incentive, it is already set in stone
	if newPPMShipment.Status == models.PPMShipmentStatusCloseoutComplete || newPPMShipment.Status == models.PPMShipmentStatusComplete {
		if oldPPMShipment.MaxIncentive == nil {
			// This PPM is past closeout and it does not have a max incentive
			return nil, apperror.NewInternalServerError(fmt.Sprintf("This PPM is past closeout, it is expected that PPM %s will have an existing max incentive", oldPPMShipment.ID.String()))
		}
		// Else, return the existing max incentive
		return oldPPMShipment.MaxIncentive, nil
	}
```

Reply:
Error response could be a bit more detailed about the internal server error, jkust gave a quick example

Reply:
LOW IMPACT - this field is only used in SSW, hotfix priority overtakes it. Existing PPMs would receive "no max incentive" on the SSW